public/index-pl.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf8">
+<title>Community Code of Conduct</title>
+<style>
+* {
+  font-family: 'helvetica neue', arial, helvetica;
+  font-weight: 200;
+}
+
+html {
+  background: #efefef;
+}
+
+body {
+  background: white;
+  color: #212121;
+  margin: 0;
+  padding-top: 60px;
+}
+
+p, h1, h2 {
+  max-width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1 {
+  font-size: 38px;
+}
+
+h2 {
+  font-size: 28px;
+  margin-top: 48px;
+}
+
+p {
+  line-height: 24px;
+  font-size: 18px;
+  text-align: justify;
+}
+
+a {
+  color: blue;
+}
+
+em {
+  font-weight: 400;
+}
+
+.footer {
+  padding: 40px 0;
+  background: #efefef;
+  margin: 0;
+  margin-top: 40px;
+  max-width: 100%;
+}
+
+.footer p {
+  text-align: left;
+}
+</style>
+</head>
+<body>
+<h1>Kodeks postępowania</h1>
+
+<p>Wszystkie uczestniczki i wszyscy uczestnicy naszej grupy zobowiązani są do zaakceptowania oraz przestrzegania niniejszego kodeksu postępowania; zawsze - w trakcie spotkań, jak i pomiędzy nimi. Jako organizatorzy, oczekujemy współpracy podczas wdrażania zasad niniejszego kodeksu, aby zapewnić otoczenie w którym wszyscy będziemy czuli się bezpiecznie.</p>
```

Comment:
It is quite binary (it is like women and men). I would write:
``` Wszytkie osoby uczestniczące w naszej grupie zobowiązane są``` 
instead of:
```Wszystkie uczestniczki i wszyscy uczestnicy naszej grupy zobowiązani są```